app/config/errors.php

```diff
@@ -136,6 +136,11 @@
         'description' => 'The current project has exceeded the maximum number of users. Please check your user limit in the Appwrite console.',
         'code' => 501,
     ],
+    Exception::USER_CONSOLE_COUNT_EXCEEDED => [
+        'name' => Exception::USER_CONSOLE_COUNT_EXCEEDED,
+        'description' => 'Sign up to the console is restricted. You can update console sign up restrictions by setting _APP_CONSOLE_WHITELIST_ROOT to "disabled" or by updating _APP_CONSOLE_WHITELIST_IPS or _APP_CONSOLE_WHITELIST_EMAILS to whitelist your IP or email.',
```

Comment:
`_APP_CONSOLE_WHITELIST_IPS` and `_APP_CONSOLE_WHITELIST_EMAILS` are for different errors (`USER_IP_NOT_WHITELISTED ` and `USER_EMAIL_NOT_WHITELISTED` respectively)